panther_utils/include/panther_utils/test/ros_test_utils.hpp

```diff
@@ -53,6 +53,30 @@ bool WaitForMsg(
   return false;
 }
 
+/**
+ * @brief Wait for ROS service future to arrive
+ *
+ * @param node ROS node that will be spin
+ * @param future Reference to the service future, it should point to a variable,
+ * that will be overwritten after service response
+ * @param timeout timeout to wait for service response
+ *
+ * @return True if message was received, false if timeout was reached
+ */
+template <typename NodeT, typename FutureT>
+bool WaitForFuture(const NodeT & node, FutureT & future, const std::chrono::nanoseconds & timeout)
```

Comment:
```suggestion
bool WaitForFuture(const NodeT & node, FutureT & future, const std::chrono::milliseconds & timeout)
```
Again milliseconds should be good enough